Code/GraphMol/Descriptors/Property.cpp

```diff
@@ -47,13 +47,14 @@ namespace Descriptors {
 
 namespace {
 void _registerDescriptors() {
-  REGISTER_DESCRIPTOR(exactmw, calcExactMW);
```

Comment:
@greglandrum Removing this broke a test, I'm going to add it back fr now.

Reply:
yeah, removing it was a mistake.